src/units/base.js

```diff
@@ -71,7 +73,14 @@ export class BaseUnit extends NamedDefaultableRepetitionRuntimeItemsImportantSet
      * @returns Boolean
      */
     isInStatus(status) {
-        return (this.status === status);
+        return this.status === status;
     }
 
+    clone(extraContext) {
+        const flowchartIDOverrideConfigAsExtraContext = {
+            flowchartId: BaseUnit.defaultFlowchartId(),
```

Comment:
Can possibly be changed to `generateFlowChartId()`

Reply:
TBD in a followup task

Reply:
https://exabyte.atlassian.net/browse/SOF-6182